server/src/main/java/p2/dao/impl/UserDAO.java

```diff
@@ -1,8 +1,26 @@
 package p2.dao.impl;
 
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+
 import p2.dao.IUserDAO;
 import p2.model.User;
+import p2.util.HibernateUtil;
+
+public class UserDAO extends GenericDAO<User> implements IUserDAO {
+
+	public User findByEmailANDPassword(String email, String password) {
+		Session session = HibernateUtil.getSession();
+		User user = null;
 
-public class UserDAO extends GenericDAO<User> implements IUserDAO { 
+		try {
+			user =  (User) session.createQuery("FROM Interest WHERE email = " + email+" AND password =" + password);
```

Comment:
use query.setParameter here (might be open to SQL injections)

Reply:
Done